src/creature.h

```diff
@@ -185,6 +185,11 @@ class Creature
         dealt_projectile_attack projectile_attack( const projectile &proj, const tripoint &target,
                                                    double total_dispersion );
 
+        /**
+         * Return the chance that a projectile attack will hit with at least the given accuracy
```

Comment:
For example here you should write:
```
/**
 *  Calculate projectile attack to hit with at least the given accuracy
 *  @return chance (0..1)
 */
```

If you use `@return`, `@param`, `@warning` doxyen extracts those separately